wcag/3.3.4.md

```diff
@@ -2,4 +2,12 @@
 
 (Level AA)
 
-Placeholder
+This applies directly as written, and as described in Intent from Understanding Success Criterion 3.3.4 replacing “web pages” with “views or screens”.
+
+With this substitution, it would read:
+
+3.3.4 Error Prevention (Legal, Financial, Data): For [screens or views] that cause [legal commitments](https://www.w3.org/TR/WCAG22/#dfn-legal-commitments) or financial transactions for the user to occur, that modify or delete [user-controllable](https://www.w3.org/TR/WCAG22/#dfn-user-controllable) data in data storage systems, or that submit user test responses, at least one of the following is true:
```

Comment:
[views] instead of [screens or views]

Maybe we can link to https://www.w3.org/TR/wcag-3.0/#dfn-views for the view definition?

`[views](https://www.w3.org/TR/wcag-3.0/#dfn-views)`

Once there is a definitive definition available we can replace all `https://www.w3.org/TR/wcag-3.0/#dfn-views` with the new URL